node/src/sub_lib/route.rs

```diff
@@ -349,7 +349,7 @@ mod tests {
     use crate::sub_lib::cryptde_null::CryptDENull;
     use crate::test_utils::make_wallet;
     use crate::test_utils::{main_cryptde, make_paying_wallet};
-    use masq_lib::test_utils::utils::TEST_DEFAULT_CHAIN;
```

Comment:
Do we really have such a silly naming that we `utils` is a file in `test_utils`? Maybe `other_utils` or `miscellaneous_utils` (oh yeah, the last one is not gonna win this).

Reply:
I don't think it's there anymore.